src/redux/constants/classifierTypes.js

```diff
@@ -1,5 +1,5 @@
 export const categories = [
-  { value: "", label: "" },
```

Comment:
Can't remove this. This array is used in all the forms. The first option is the default option of the dropdown menu and has to be an empty string to force users to choose from one of the actual categories

Reply:
oh shoot okay I wasnt aware